docs/products/clickhouse/getting-started.rst

```diff
@@ -0,0 +1,45 @@
+Getting started
+===============
+
+The first step in using Aiven for ClickHouse is to create a service. You can do this in the `Aiven web console <https://console.aiven.io/>`_ or with the `Aiven CLI <https://github.com/aiven/aiven-client>`_.
+
+
+Create a ClickHouse service in the Aiven web console
+----------------------------------------------------
+
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+
+2. On the *Services* page, click **Create a new service**.
+
+3. Select the main properties for your service:
+
+   a. Select **ClickHouse** as the service type.
+
+   b. Select the cloud provider and region that you want to run your service on.
+
+      .. note:: 
+	      The pricing for the same service may vary between different providers and regions. The service summary on the right side of the console shows you the pricing for your selected options.
+
+   c. Select a service plan.
+
+      This defines the number of nodes, storage, memory, CPU, and backup support for the service.
+
+   d. Enter a name for your service.
+
+      A random name is provided by default, but you can enter a more recognizable name to distinguish it from other services.
+
+
+4. Click **Create Service** under the summary on the right side of the console.
+
+   This brings you back to the **Services** view. Your new service is listed with a status indicator to show that it is being created.
+
+5. Click the service name.
+
+   The *Overview* page for the service opens.
+
+   This view shows you the connection parameters for your service, its current status, and the configuration options.
+
+   The status is *Rebuilding* while the service is being created for you. Once the service is ready, the status changes to *Running*. While services typically start up in a couple of minutes, the performance varies between cloud providers and regions, and it may take longer in some circumstances.
+
+
```

Comment:
I think we should revisit this page, creating a service does not get me started.

How about including an instruction to create a database, and showing an example of connecting with the docker version of the client command? Then leading people on immediately to the sample data, code samples if we have them, and other resources.

I don't see a reason to duplicate the "creating a service" page unless there's something different in the setup for this particular service that we want to mention. For example, linking to the generic create, and recommending some configuration or replication values seems more valuable to me.

Reply:
Sure, that's now updated ;)